cdap-api/src/main/java/co/cask/cdap/api/schedule/ScheduleConfigurer.java

```diff
@@ -32,9 +32,9 @@
 
   ScheduleConfigurer delayRun(long delayMillis);
 
-  ScheduleConfigurer setTimeRange(int startHour, int endHour);
+  ScheduleConfigurer setTimeRange(int startHour, int startMinute, int endHour, int endMinute);
```

Comment:
It will just conflict with the changes I made to the same class as a part of https://github.com/caskdata/cdap/pull/8765